src/main/java/net/malisis/core/inventory/MalisisInventory.java

```diff
@@ -453,7 +453,7 @@ public void onSlotChanged(MalisisSlot slot)
 	 * Called when this {@link MalisisInventory} is opened.
 	 */
 	@Override
-	public void openInventory()
+	public void openChest()
```

Comment:
The previous name made 100% more sense but stable_12 got the bad name. I'd love to change it but dunno how to invoke a "stable_13" to be built and doubt Lex would sense 1.7 is now in maintaince mode.